bakerydemo/templates/includes/footer.html

```diff
@@ -2,26 +2,26 @@
 
 <div class="container" role="contentinfo">
```

Comment:
Switch to footer tag?

Reply:
Having switched to the footer tag, I removed the role attribute, as this is no longer necessary. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role